### PR TITLE
Sprint37 executionid

### PR DESCRIPTION
**Pull request by nilson-lagos**
executionTask or/and github covered by this fix
https://app.asana.com/0/1121509287610684/1156903668470665

Answer the following questions:
Did you include or update the unit tests?
yes
Did you execute the npm tests ?
yes
Did you test it in both environments (dev/prod)?
yes
if you update the routes... Did you update the sample.txt ?
no routes updated

### CODE REVIEW

File: key.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+vcajqkxav8icq1rcheyz4r-rupgajpy9v7qmm0ximd3v-lb5481whf5qjfxec6p7xss-tb0ipdtg8n1se4klq7f1p
```
Comment (renan-ua) [inline key.pub]: remove this key

File: mkcli.py [hunk 2]
```diff
@@ -61,18 +62,20 @@ def run(args):
...
   route = 'src/test/groovy'
   browser = args.browser
+  executionNumber = None
   #Exit code to report at circleci
   exitCode = 1
   #Check if we received a browser and get the string for the gradlew task command
   browserName = getBrowserName(browser)
   muuktestRoute = 'https://portal.muuktest.com:8081/'
-  supportRoute = 'https://testing.muuktest.com:8082/'
+  supportRoute = 'https://portal.muuktest.com:8082/'
```
Comment (renan-ua) [inline mkcli.py]: support route is testing

File: mkcli.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import xml.etree.ElementTree
 from time import strftime
 from mkcloud import gatherScreenshots, resizeImages
-#import ssl
+# import ssl
```
Comment (renan-ua) [inline mkcli.py]: fix space

File: mkcli.py [hunk 4]
```diff
@@ -91,7 +94,7 @@ def run(args):
     key_file = open(path,'r')
     key = key_file.read()
     r = requests.post(muuktestRoute+"generate_token_executer", data={'key': key})
-    #r = requests.post(muuktestRoute+"generate_token_executer", data={'key': key}, verify=False)
+    # r = requests.post(muuktestRoute+"generate_token_executer", data={'key': key}, verify=False)
```
Comment (renan-ua) [inline mkcli.py]: fix sapce for all comments applied

File: mkcli.py [hunk 5]
```diff
@@ -109,6 +112,10 @@ def run(args):
     # #Delete the old files
     if os.path.exists("test.rar"):
       os.remove('test.rar')
+    for item in os.listdir('.'):
+      if item.endswith(".mp4"):
+        os.remove(item)
+
```
Comment (renan-ua) [inline mkcli.py]: this seems unrelated to this fix

File: mkcli.py [hunk 6]
```diff
@@ -152,6 +159,13 @@ def run(args):
       # Unzip the file // the library needs the file to end in .rar for some reason
       shutil.unpack_archive('test.zip', extract_dir=route, format='zip')
 
+      try:
+        execFile = open('src/test/groovy/executionNumber.execution', 'r')
```
Comment (renan-ua) [inline mkcli.py]: before you try to open, verify the file does exist ...

File: mkcli.py [hunk 7]
```diff
@@ -182,7 +196,7 @@ def run(args):
 
         testsExecuted = gatherFeedbackData(browserName)
         url = muuktestRoute+'feedback/'
-        values = {'tests': testsExecuted, 'userId': userId, 'browser': browserName}
+        values = {'tests': testsExecuted, 'userId': userId, 'browser': browserName,'executionNumber': int(executionNumber)}
```
Comment (renan-ua) [inline mkcli.py]: what happen if executionNumber is None because it was not provided?

File: mkcli.py [hunk 8]
```diff
@@ -192,7 +206,7 @@ def run(args):
         try:
           if filesData != {}:
             requests.post(muuktestRoute + 'upload_cloud_steps_images/', headers=hed, files = filesData)
-            #requests.post(muuktestRoute + 'upload_cloud_steps_images/', data={'cloudKey': cloudKey}, headers=hed, files = filesData,  verify=False)
+            # requests.post(muuktestRoute + 'upload_cloud_steps_images/', headers=hed, files = filesData,  verify=False)
```
Comment (renan-ua) [inline mkcli.py]: leave the line as it was (including cloudkey data ).... 
             #requests.post(muuktestRoute + 'upload_cloud_steps_images/', data={'cloudKey': cloudKey}, headers=hed, files = filesData,  verify=False)

File: mkcli.py (hunk 7 above)
Comment (nilson-lagos) [reply mkcli.py]: it will send NaN to the BE, execution wont be found and the feedback won't be saved
